test/mlir/conversion/onnx_to_torch/gemm/gem2_transB_one.onnx.mlir

```diff
@@ -0,0 +1,25 @@
+//RUN: onnx-mlir --EmitONNXIR --run-torch-pass %s -o=%t >/dev/null && cat %t.onnx.mlir | FileCheck -v %s
```

Comment:
@anushree1998 can you please rename the testcases?

For example, rename it to:
`gemm_transAB.onnx.mlir` like mentioned in earlier review, lets not name with numbers.

Reply:
We have made the changes required in the latest commit.

Reply:
Please remove the `_zero` and `_one`